lib/StartInfo.cpp

```diff
@@ -58,13 +58,47 @@ bool PlayerSettings::isControlledByHuman() const
 	return !connectedPlayerIDs.empty();
 }
 
+std::vector<std::string> StartInfo::getPlayerNames() const
+{
+	std::vector<std::string> playerNames;
+	for(PlayerColor player(0); player < PlayerColor::PLAYER_LIMIT; ++player)
+	{
+		if(!playerInfos.count(player))
+			continue;
+
+		auto playerInfo = playerInfos.at(player);
+		if(playerInfo.isControlledByHuman())
+		{
+			playerNames.push_back(playerInfo.name);
+		}
+	}
+
+	return playerNames;
+}
+
+uint8_t StartInfo::getHumanPlayersCount() const
+{
+	uint8_t NoOfHumanPlayers = 0;
```

Comment:
Local variables start with non-capital in vcmi code

Reply:
renamed to humanPlayersCount